crates/core/src/delta_datafusion/mod.rs

```diff
@@ -542,6 +542,7 @@ impl<'a> DeltaScanBuilder<'a> {
             .filter
             .map(|expr| context.create_physical_expr(expr, &df_schema).unwrap());
 
+        // dbg!(logical_filter.clone());
```

Comment:
We prolly don't need these now ?

Reply:
I was debugging a regression caused by the kernel bump. There is something wrong with the statistics being passed to Datafusion